doc/ansible/dev/developer_guide.md

```diff
@@ -0,0 +1,234 @@
+changed: [localhost]
+
+PLAY RECAP *********************************************************************************
+localhost                  : ok=2    changed=1    unreachable=0    failed=0
+
+```
+
+Check that the namespace was deleted:
+```bash
+$ kubectl get namespace
+NAME          STATUS    AGE
+default       Active    28d
+kube-public   Active    28d
+kube-system   Active    28d
+```
+## Watches file
+
+The Operator expects a mapping file, which lists each GVK to watch and the
+corresponding path to an Ansible role or playbook, to be copied into the
+container at a predefined location: /opt/ansible/watches.yaml
+
+Example:
+``Dockerfile
+COPY watches.yaml /opt/ansible/watches.yaml
+``
+
+The Watches file format is yaml and is an array of objects. The object has
+mandatory fields:
+
+**version**:  The version of the Custom Resource that you will be watching.
+
+**group**:  The group of the Custom Resource that you will be watching.
+
+**kind**:  The kind of the Custom Resource that you will be watching.
+
+**playbook**:  This is the path to the playbook that you have added to the
+container. This playbook is expected to be simply a way to call roles. This
+field is mutually exclusive with the "role" field.
+
+**role**:  This is the path to the role that you have added to the container.
+For example if your roles directory is at `/opt/ansible/roles/` and your role
+is named `busybox`, this value will be `/opt/ansible/roles/busybox`. This field
+is mutually exclusive with the "playbook" field.
+
+Example specifying a role:
+
+```yaml
+---
+- version: v1alpha1
+  group: foo.example.com
+  kind: Foo
+  role: /opt/ansible/roles/Foo
+```
+
+### Using playbooks in watches.yaml
+
+By default, `operator-sdk new --type ansible` sets `watches.yaml` to execute a
+role directly on a resource event. This works well for new projects, but with a
+lot of Ansible code this can be hard to scale if we are putting everything
+inside of one role. Using a playbook allows the developer to have more
+flexibility in consuming other roles and enabling more customized deployments
+of their application. To do this, modify `watches.yaml` to use a playbook
+instead of the role:
+```yaml
+---
+- version: v1alpha1
+  group: foo.example.com
+  kind: Foo
+  playbook: /opt/ansible/playbook.yml
+```
+
+Modify `tmp/build/Dockerfile` to put `playbook.yml` in `/opt/ansible` in the
+container in addition to the role (`/opt/ansible` is the `HOME` environment
+variable inside of the Ansible Operator base image):
+```Dockerfile
+FROM quay.io/water-hole/ansible-operator
+
+COPY roles/ ${HOME}/roles
+COPY playbook.yaml ${HOME}/playbook.yaml
+COPY watches.yaml ${HOME}/watches.yaml
+```
+
+Alternatively, to generate a skeleton project with the above changes, a
+developer can also do:
+```bash
+$ operator-sdk new --type ansible --kind Foo --api-version foo.example.com/v1alpha1 foo-operator --generate-playbook
+```
+
+## Testing an Ansible operator locally
```

Comment:
This section is dependent on a solution for https://github.com/ansible/ansible-runner/issues/152

Whether or not I set the environment variable as a short term fix is outside the scope of this document